include/networkit/graph/GraphTools.hpp

```diff
@@ -129,13 +129,86 @@ std::pair<node, node> size(const Graph &G) noexcept;
 double density(const Graph &G) noexcept;
 
 /**
- * Returns the volume (sum of the out-degree of all nodes) of the graph.
+ * Returns the volume of the input graph.
  *
  * @param G The input graph.
  * @return Volume of the graph.
+ * 
  */
 double volume(const Graph &G);
 
+/**
+ * Returns the volume (sum of the out-degree of all nodes) of a range
+ * of nodes.
+ *
+ * @param G The input graph.
+ * @param first Iterator for the starting node in the set.
```

Comment:
Maybe it is clearer `@param first,last The range of nodes to insert` as they do in cppreference (e.g., https://en.cppreference.com/w/cpp/container/vector/insert).